components/browser/thumbnails/src/test/java/mozilla/components/browser/thumbnails/utils/ThumbnailDiskCacheTest.kt

```diff
@@ -76,4 +79,17 @@ class ThumbnailDiskCacheTest {
         data = cache.getThumbnailData(testContext, request)
         assertNull(data)
     }
+
+    @Test
+    fun `Clearing bitmap from disk catch IOException`() {
+        val cache = ThumbnailDiskCache()
+        val lruCache: DiskLruCache = mock()
+        cache.thumbnailCache = lruCache
+
+        `when`(lruCache.delete()).thenAnswer { throw IOException("test") }
```

Comment:
nit: There's also `thenThrow()`.

Reply:
Oops, I was trying out something with this test and forgo to revert it. :)